CodeEdit/Documents/WorkspaceDocument.swift

```diff
@@ -368,6 +380,89 @@ import TabBar
 
         super.close()
     }
+
+    /// Determines the windows should be closed.
+    ///
+    /// This method iterates all editied documents If there are any editied documents.
+    ///
+    /// A panel giving the user the choice of canceling, discarding changes, or saving is presented while iteration.
+    ///
+    /// If the user chooses cancel on the panel, iteration is breaked.
```

Comment:
Grammatical error: "breaked" should be "broken" #picky

Reply:
I fixed this grammatical error.